linkage/demo_notebooks/2022_10_21a_prl_splink_test_part_1.ipynb

```diff
@@ -0,0 +1,3242 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        training_sessions[rule] = linker.estimate_parameters_using_expectation_maximisation(training_blocking_rules[1])
</pre><p>The fact that you used [1] at the end of this line is why all three are the same.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research_prl/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>